docs/src/components/Container/Helper.js

```diff
@@ -0,0 +1,6 @@
+export function onScrollHandler() {
+    var element = document.getElementById('container');
+    var tabsDimension = element?.getBoundingClientRect();
+    const isPinned =  tabsDimension?.top < 50 ? true : false;
```

Comment:
Instead of using fixed number **50**, please calculate the height of header and then use it